### PR TITLE
march ide release note

### PR DESCRIPTION
**Pull request by mirnawong1**
Release note [request](https://www.notion.so/dbtlabs/IDE-Updates-March-7be7d144f2da49fbb7d85cd61bb86b43?pvs=4) highlight March IDE [updates](https://getdbt.slack.com/archives/C03SAHKKG2Z/p1679942726846429). pr includes:
- ide changes
- addition of blog link
- tweaks to past ide release notes (add 'release note' in description to be even more clear) 

## Checklist

- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding new pages (delete if not applicable):
- [ ] Add page to `website/sidebars.js`
- [ ] Provide a unique filename for the new page

Removing or renaming existing pages (delete if not applicable):
- [ ] Remove page from `website/sidebars.js`
- [ ] Add an entry `_redirects`
- [ ] [Ran link testing](https://github.com/dbt-labs/docs.getdbt.com#running-the-cypress-tests-locally) to update the links that point to the deleted page

### CODE REVIEW

File: website/docs/docs/dbt-versions/release-notes/10-Mar-2023/mar-ide-updates.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+- When the Format button fails, a toast notification will display a syntax error.
+- The editor now has the option to toggle minimap/word-wrap via right-click.
+- The history drawer now displays elapsed time more in real-time and in s/m/h increments.
+- When deleting development environments, the delete modal will now warn users that any uncommitted changes will be lost.
+- The context for the Git button has been adjusted to show that it will link to an external site (such as GitHub or GitLab) when users create a pull request.
+
+## Bug fixes
+
+- The IDE now displays an error message when the git repository is not reachable. Previously, it failed silently.
+- On small screens, the **Split** button is now hidden.
```
Comment (bossnunta) [inline website/docs/docs/dbt-versions/release-notes/10-Mar-2023/mar-ide-updates.md]: let's actually remove this line about split button -- it's not that relevant

File: website/docs/docs/dbt-versions/release-notes/10-Mar-2023/mar-ide-updates.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+- The kebab menu is now visible when the invocation history drawer is open. Previously, it wasn't showing.
+- DAGs are now updated/populated consistently. Previously, it occasionally failed.
+- The purple highlight for DAG selection is now consistent across files. Previously, it was inconsistent.
+- Users can now rename files back to their original name. Previously, this wasn't possible.
+- The link to the IDE from the project setup page has been corrected.
+- The IDE no longer has issues with single-space file names.
+- Adding invalid characters in the sub-directory config no longer causes the IDE to fail.
+- YML schema validation now consistently passes. Previously, it failed occasionally.
```
Comment (bossnunta) [inline website/docs/docs/dbt-versions/release-notes/10-Mar-2023/mar-ide-updates.md]: This is more like
"We've address the issue where YML autocompletion did not trigger all the time."

**Review by bossnunta — COMMENTED** [review]
thanks mirna -- minor correction but other lgtm

File: website/docs/docs/dbt-versions/release-notes/10-Mar-2023/mar-ide-updates.md (hunk 2 above)
Comment (mirnawong1) [reply website/docs/docs/dbt-versions/release-notes/10-Mar-2023/mar-ide-updates.md]: ah gotcha! what about:

"YML autocomplete triggers consistently now. Previously, it occasionally didn't trigger."?

File: website/docs/docs/dbt-versions/release-notes/10-Mar-2023/mar-ide-updates.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+---
+
+To continue improving your [Cloud IDE](https://docs.getdbt.com/docs/cloud/develop-in-the-cloud) development experience, the dbt Labs team continue to work on adding new features, fixing bugs, and increasing reliability ✨.
+
+Read more about the [upcoming improvements to the Cloud IDE](https://www.getdbt.com/blog/improvements-to-the-dbt-cloud-ide/) and stay up-to-date with [IDE-related changes](https://docs.getdbt.com/tags/ide).
+
+
+## New features 
+
+- You can now commit and revert individual files under **Version Control**.
```
Comment (matthewshaver) [inline website/docs/docs/dbt-versions/release-notes/10-Mar-2023/mar-ide-updates.md]: Just a small suggestion - The assumption of the new features is that it's something they can do now which previously could not be done.
```suggestion
- Commit and revert individual files under **Version Control**.
```

File: website/docs/docs/dbt-versions/release-notes/10-Mar-2023/mar-ide-updates.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+- You can now use the [command palette](/docs/cloud/develop-in-the-cloud#cloud-ide-features) to invoke common complex dbt commands, such as resuming from the last failure.
```
Comment (matthewshaver) [inline website/docs/docs/dbt-versions/release-notes/10-Mar-2023/mar-ide-updates.md]:
```suggestion
- Use the [command palette](/docs/cloud/develop-in-the-cloud#cloud-ide-features) to invoke common complex dbt commands, such as resuming from the last failure.
```

File: website/docs/docs/dbt-versions/release-notes/10-Mar-2023/mar-ide-updates.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+- Under the git dropdown, you can now create PRs even when there are uncommitted changes.
```
Comment (matthewshaver) [inline website/docs/docs/dbt-versions/release-notes/10-Mar-2023/mar-ide-updates.md]:
```suggestion
- Create PRs even when there are uncommitted changes (under the **git** dropdown).
```

File: website/docs/docs/dbt-versions/release-notes/10-Mar-2023/mar-ide-updates.md [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+- You will now see more autocomplete suggestions when editing a YML file; this is powered by [dbt-jsonschema](https://github.com/dbt-labs/dbt-jsonschema).
```
Comment (matthewshaver) [inline website/docs/docs/dbt-versions/release-notes/10-Mar-2023/mar-ide-updates.md]:
```suggestion
- The IDE will display more autocomplete suggestions when editing a YML file, powered by [dbt-jsonschema](https://github.com/dbt-labs/dbt-jsonschema).
```

File: website/docs/docs/dbt-versions/release-notes/10-Mar-2023/mar-ide-updates.md [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+- The file tree now has additional options in the right-click menu, such as Copy model as ref or Copy file path.
+- The DAG view has been adjusted to a default of `2+model+2`.
+- A lineage selector has been implemented in the DAG/lineage sub-tab.
+- You can now edit directly in the git diff view, located in the right pane.
```
Comment (matthewshaver) [inline website/docs/docs/dbt-versions/release-notes/10-Mar-2023/mar-ide-updates.md]:
```suggestion
- Edit directly in the git diff view located in the right pane.
```

File: website/docs/docs/dbt-versions/release-notes/10-Mar-2023/mar-ide-updates.md [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+- A new onboarding flow guide is now available.
+
+## Product refinements 
+
+- The DAG selector now uses `name` instead of `file_uri` to build selectors.
+- The DAG is now vertically centered under the new Selector Input element 
+- sqlfmt has been upgraded to v0.17.0.
+- When the Format button fails, a toast notification will display a syntax error.
+- The editor now has the option to toggle minimap/word-wrap via right-click.
+- The history drawer now displays elapsed time more in real-time and in s/m/h increments.
```
Comment (matthewshaver) [inline website/docs/docs/dbt-versions/release-notes/10-Mar-2023/mar-ide-updates.md]:
```suggestion
- The history drawer displays elapsed time in real-time and s/m/h increments.
```

**Review by matthewshaver — APPROVED** [review]

**Pull request merged**